src/v2/Apps/_Preferences2/PreferencesApp.tsx

```diff
@@ -5,30 +5,101 @@ import {
   GridColumns,
   Separator,
   Text,
+  useToasts,
 } from "@artsy/palette"
-import { Formik } from "formik"
+import { isEmpty, camelCase, snakeCase } from "lodash"
+import { FC } from "react"
+import { Form, Formik } from "formik"
+import { createFragmentContainer, graphql } from "react-relay"
+import {
+  PreferencesApp_viewer,
+  SubGroupStatus,
+} from "v2/__generated__/PreferencesApp_viewer.graphql"
+import { useEditNotificationPreferences } from "./useEditNotificationPreferences"
+
+const NOTIFICATION_FIELDS = {
+  recommendedByArtsy: false,
+  artWorldInsights: false,
+  productUpdates: false,
+  guidanceOnCollecting: false,
+  customAlerts: false,
+}
+
+interface PreferencesAppProps {
+  viewer?: PreferencesApp_viewer
+}
+
+interface FormValuesForNotificationPreferences {
+  recommendedByArtsy: boolean
+  artWorldInsights: boolean
+  productUpdates: boolean
+  guidanceOnCollecting: boolean
+  customAlerts: boolean
+}
+
+const parseViewer = viewer => {
+  return viewer?.notificationPreferences
+    .filter(preference =>
+      Object.keys(NOTIFICATION_FIELDS).includes(camelCase(preference.name))
+    )
+    .reduce((object, preference) => {
+      object[camelCase(preference.name)] =
+        preference.status === "SUBSCRIBED" ? true : false
+      return object
+    }, {})
+}
```

Comment:
I'm sure this could be improved, but it's not very expensive. `viewer.notificationPreferences` is probably going to be fewer than 10 objects for the foreseeable future.